src/libraries/Native/native-binplace.proj

```diff
@@ -1,29 +1,29 @@
-<Project DefaultTargets="Build">
-
-  <Import Project="..\Directory.Build.props" />
-  <Target Name="Build" DependsOnTargets="BinPlace" />
-  <Target Name="Clean" />
-  <Target Name="Rebuild" DependsOnTargets="Clean;Build" />
-
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.targets))" />
+<Project>
```

Comment:
I wonder if we could have an SDK that just has empty Build targets that we could just hook off of, and avoid explicit imports.

Reply:
Could we use Microsoft.Build.Traversal?

Reply:
Actually you would want to use https://github.com/microsoft/MSBuildSdks/tree/master/src/NoTargets here but I hit some issues with our multi-targeting approach and reverted my changes. I can re-investigate in my follow-up work.